### PR TITLE
task: Add an OkHttpMetricsSender

### PR DESCRIPTION
**Pull request by chriswk**
So, having added an OkHttpFeatureFetcher in a previous PR, this PR adds a metricssender which uses OkHttp.

### CODE REVIEW

**Comment by thomasheartman** [comment]
@chriswk This PR has been marked as stale: do we still want to get it in or should we close it?

File: src/main/java/io/getunleash/metric/JavaHttpMetricsSender.java [hunk 1]
```diff
@@ -25,7 +23,7 @@ public class UnleashMetricsSender {
     private final URL clientRegistrationURL;
     private final URL clientMetricsURL;
 
-    public UnleashMetricsSender(UnleashConfig unleashConfig) {
+    public JavaHttpMetricsSender(UnleashConfig unleashConfig) {
```
Comment (sighphyre) [inline src/main/java/io/getunleash/metric/JavaHttpMetricsSender.java:26]: Very optional nit: Happy with the name here? The Java feels a bit redundant in a Java program. DefaultMetricsSender maybe?

**Review by sighphyre — APPROVED** [review]
Looks good, I've put down an optional thing around naming but I'm not going to block this if you're happy with the name. Let's get this beast in!

File: src/main/java/io/getunleash/metric/JavaHttpMetricsSender.java (hunk 1 above)
Comment (chriswk) [reply src/main/java/io/getunleash/metric/JavaHttpMetricsSender.java:26]: Well, it is using java.net.http :P But agreed that it feels redundant. Default is fair. I'll do the rename

**Pull request merged**